### ISSUE TITLE
Lesson 4 pip installation

### ISSUE THREAD

**Issue by Sheep-Code** [smartcontractkit/full-blockchain-solidity-course-py#1328]
I've installed python 3.10 but there is a problem with pip. In Visual Studio I get this error when typing pip --version "zsh: command not found: pip" Could be something small I am very new to all this although I have been looking for awhile now and cant find the solution any help?

**Comment by Sheep-Code** [comment]
Yes I can run python --version and get "Python 3.10.3" which is why I am confused because like you said pip should be downloaded with python now. Also when I downloaded python version 3.10.3 it was initially stuck in 2 something I dont remember exactly the version. But I thought maybe that was why pip wasn't showing up so I switched to version 3.10.3 and I was still getting "zsh: command not found: pip".

**Comment by sillytechie** [comment]
I had the same issue.
After some googling realized that I need to use "python3" where the tutorial says "python" and "pip3" where the tutorial uses "pip" 
hope that helps
